core/templates/dev/head/domain/suggestion/SuggestionThreadObjectFactorySpec.js

```diff
@@ -51,7 +51,9 @@ describe('Suggestion thread object factory', function() {
         cmd: 'edit_state_property',
         property_name: 'content',
         state_name: 'state_1',
-        new_value: 'new suggestion content',
+        new_value: {
+          html: 'new suggestion content'
+        },
         old_value: 'old suggestion content'
```

Comment:
Is it correct that old_value and new_value are formatted differently here?

Reply:
Oops.. fixed